File: Examples/CSharp/Slides/Notes/RemoveNotesAtSpecificSlide.cs
using Aspose.Slides.Export;

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https://docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http://www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.Slides.Examples.CSharp.Slides.Notes
{
    class RemoveNotesAtSpecificSlide
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Slides_Presentations_Notes();

            // ExStart:RemoveNotesAtSpecificSlide
            // Instantiate a Presentation object that represents a presentation file 
            Presentation presentation = new Presentation(dataDir + "AccessSlides.pptx");

            // Removing notes of first slide
            INotesSlideManager mgr = presentation.Slides[0].NotesSlideManager;
            mgr.RemoveNotesSlide();

            // ExEnd:RemoveNotesAtSpecificSlide
            // Save presentation to disk
            presentation.Save(dataDir + "RemoveNotesAtSpecificSlide_out.pptx", SaveFormat.Pptx);
        }
    }
}

File: Examples/VisualBasic/Slides/Notes/RemoveNotesAtSpecificSlide.vb
Imports System
Imports Aspose.Slides.Export

'
'This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
'when the project is build. Please check https:// Docs.nuget.org/consume/nuget-faq for more information. 
'If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http://www.aspose.com/downloads, 
'install it and then add its reference to this project. For any issues, questions or suggestions 
'please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
'

Namespace Aspose.Slides.Examples.VisualBasic.Slides.Notes
    Class RemoveNotesAtSpecificSlide
        Public Shared Sub Run()
            ' ExStart:RemoveNotesAtSpecificSlide
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Slides_Presentations_Notes()

            ' Instantiate a Presentation object that represents a presentation file 
            Dim presentation As New Presentation(dataDir & Convert.ToString("AccessSlides.pptx"))

            ' Removing notes of first slide
            Dim mgr As INotesSlideManager = presentation.Slides(0).NotesSlideManager
            mgr.RemoveNotesSlide()

            ' Save presentation to disk
            presentation.Save(dataDir & Convert.ToString("RemoveNotesAtSpecificSlide_out.pptx"), SaveFormat.Pptx)
            ' ExEnd:RemoveNotesAtSpecificSlide
        End Sub
    End Class
End Namespace